ena-submission/test/text_project_request.xml

```diff
@@ -1,8 +1,8 @@
 <PROJECT_SET>
-    <PROJECT center_name="Test institution" alias="2:Test organism:Test suffix">
-        <NAME>Test scientific name: Genome sequencing by Test group, Test institution</NAME>
-        <TITLE>Test scientific name: Genome sequencing by Test group, Test institution, Basel, Switzerland</TITLE>
-        <DESCRIPTION>Automated upload of Test scientific name sequences submitted by Test group, Test institution, Basel, Switzerland to Loculus</DESCRIPTION>
+    <PROJECT center_name="Test institution with funny characters \ / &apos;&quot;!@#$%^&amp;*()" alias="2:Test organism:Test suffix">
```

Comment:
love the test but Im a bit surprised that `'` becomes `&apos;` in the xml? Is there a way to avoid this? I'm worried ENA will see center_name=`My team's project` in teh assembly and center_name=`My team&apos;s project` in the bioproject and not link them

Reply:
Yep - it's just escaping :)